.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gomod" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+      time: "09:30"
+      timezone: "America/New_York"
+    labels:
+      - "safe to test"
```

Comment:
Does the safe to test label help here somehow? I don't know much about dependabot's configuration tbh.

Reply:
It will just automatically tag the PR with safe to test so we don't have to do it. It automatically signs off the commits too so this will be a pretty smooth process except for verifying the tests and merging